notepad/index.html

```diff
@@ -5,9 +5,9 @@
   <meta charset="utf-8">
   <title>Notepad</title>
   <script src="https://docs.getgrist.com/grist-plugin-api.js"></script>
-  <link href="//cdn.quilljs.com/1.3.6/quill.snow.css" rel="stylesheet">
-  <link href="//cdn.quilljs.com/1.3.6/quill.bubble.css" rel="stylesheet">
-  <script src="//cdn.quilljs.com/1.3.6/quill.min.js"></script>
+  <link href="https://cdn.jsdelivr.net/npm/quill@2.0.2/dist/quill.snow.css" rel="stylesheet">
```

Comment:
Do we need to update the editor? The major version has changed, so it might work differently.

Reply:
Unfortunately, `getSemanticHTML` is not available in `v1`.